src/test/javascript/spec/component/competencies/competency-form.component.spec.ts

```diff
@@ -61,6 +61,17 @@ describe('CompetencyFormComponent', () => {
         const competencyService = TestBed.inject(CompetencyService);
         const getAllTitlesSpy = jest.spyOn(competencyService, 'getCourseCompetencyTitles').mockReturnValue(of(new HttpResponse({ body: ['test'], status: 200 })));
 
+        const competencyOfResponse: Competency = {};
```

Comment:
Is it possible to set the values `competencyOfResponse.id` and `competencyOfResponse.title` in the constructor already? In the same way as with the `response`?

```suggestion
        const competencyOfResponse: Competency = new competencyOfResponse({
                id: 1,
                title: 'test',
        });
```

Reply:
It is not completely possible, but we can do it like this:
```suggestion
        const competencyOfResponse: Competency = {
                id: 1,
                title: 'test',
        };
```